website/source/docs/secrets/ad/index.html.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: "docs"
+page_title: "Active Directory - Secrets Engines"
+sidebar_current: "docs-secrets-active-directory"
+description: |-
+  The Active Directory secrets engine for Vault generates passwords dynamically based on
+  roles.
+---
+
+# Active Directory Secrets Engine
+
+The Active Directory (AD) secrets engine is a plugin residing [here](https://github.com/hashicorp/vault-plugin-secrets-active-directory).
+
+The AD secrets engine rotates AD passwords dynamically,
+and is designed for a high-load environment where many instances may be accessing
+a shared password simultaneously. With a simple set up and a simple creds API,
+it doesn't require instances to be manually registered in advance to gain access. 
+As long as access has been granted to the creds path via a method like 
+[AppRole](https://www.vaultproject.io/api/auth/approle/index.html), they're available.
+
+Passwords are lazily rotated based on preset TTLs and can have a length configured to meet 
+your needs.
+
+## A Note on Escaping
+
+**It is up to the administrator** to provide properly escaped DNs. This
+includes the user DN, bind DN for search, and so on.
+
+The only DN escaping performed by this method is on usernames given at login
+time when they are inserted into the final bind DN, and uses escaping rules
+defined in RFC 4514.
+
+Additionally, Active Directory has escaping rules that differ slightly from the
+RFC; in particular it requires escaping of '#' regardless of position in the DN
+(the RFC only requires it to be escaped when it is the first character), and
+'=', which the RFC indicates can be escaped with a backslash, but does not
+contain in its set of required escapes. If you are using Active Directory and
+these appear in your usernames, please ensure that they are escaped, in
+addition to being properly escaped in your configured DNs.
+
+For reference, see [RFC 4514](https://www.ietf.org/rfc/rfc4514.txt) and this
+[TechNet post on characters to escape in Active
+Directory](http://social.technet.microsoft.com/wiki/contents/articles/5312.active-directory-characters-to-escape.aspx).
+
+## Quick Setup
+
+Most secrets engines must be configured in advance before they can perform their
+functions. These steps are usually completed by an operator or configuration
+management tool.
+
+0. Add the AD secrets plugin to your catalogue:
```

Comment:
Do we not plan on bundling this plugin?

Reply:
Good question, I'm not certain, I'll ask.

Reply:
Updated this because it will be bundled. :-)